src/components/application_manager/include/application_manager/smart_object_keys.h

```diff
@@ -280,6 +280,7 @@ const char location_name[] = "locationName";
 const char location_description[] = "locationDescription";
 const char address_lines[] = "addressLines";
 const char phone_number[] = "phoneNumber";
+const char number[] = "number";
```

Comment:
@dev-gh We have 2 numbers: `phone_number` & just `number`. Do we need them both? What is the difference between them?